src/Scene/GlobeUpdater.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Generated On: 2016-11-15
+ * Class: GlobeUpdater
+ * Description: Updaters for Globes.
+ */
+
+import NodeProcess from 'Scene/NodeProcess';
+import Quadtree from 'Scene/Quadtree';
+import TreeUpdater from 'Scene/TreeUpdater';
+import Layer from 'Scene/Layer';
+import DefaultUpdater from 'Scene/DefaultUpdater';
+import MobileMappingLayer from 'MobileMapping/MobileMappingLayer';
+import MobileMappingUpdater from 'Scene/MobileMappingUpdater';
+
+
+function GlobeUpdater(ellipsoid) {
+    this.treeUpdater = new TreeUpdater(new NodeProcess(ellipsoid));
```

Comment:
GlobeUpdate **is** a TreeUpdater. `defaultUpdater` and `mobileMappingUpdater` should go away